lib/report-translator.js

```diff
@@ -41,7 +42,10 @@ function normalizeMultiArgReportCall() {
 
     // If there is one argument, it is considered to be a new-style call already.
     if (arguments.length === 1) {
-        return arguments[0];
+
+        // Shallow clone the object to avoid surprises
+        // if reusing the descriptor
```

Comment:
We've recently enabled a rule in our codebase related to comments spanning multiple lines. I would suggest rebasing and rerunning `npm run lint` to be sure this won't be flagged.